app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/api/templates/Readme.md

```diff
@@ -0,0 +1 @@
+# Readme
```

Comment:
@anikbrazeau 

Do we have any content for this Readme yet?

If not we can look to merge as is and update it later.